pages/portfolio.html

```diff
@@ -4,9 +4,36 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
 <body>
+  <h1><b>Projects</b></h1>
```

Comment:
Consider wrapping this in a `header`tag (note, there was a project requirement to use `header` on each page.)